Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/ui/onboarding/OnboardingFragmentTest.kt

```diff
@@ -54,11 +53,10 @@ class OnboardingFragmentTest : BaseUITest() {
     @Test
     fun capture_screenshot() {
         launchFragmentInContainer2<OnboardingFragment>()
-        Thread.sleep(SCREENSHOT_DELAY_TIME)
-        Screengrab.screenshot(OnboardingFragment::class.simpleName)
+        takeScreenshot<OnboardingFragment>()
 
         onView(withId(R.id.onboarding_easy_language)).perform(scrollTo())
-        Screengrab.screenshot(OnboardingFragment::class.simpleName.plus("2"))
+        takeScreenshot<OnboardingFragment>("2", delay = 0L)
```

Comment:
This also needs default delay otherwise screenshots look the same

Reply:
I don't think so, there was no time delay before here. Look into the changed code

Reply:
Delay was introduced just to give espresso the time to display the layout. in some cases the layout is already displayed and you need to perform an action

Reply:
<img width="527" alt="image" src="https://user-images.githubusercontent.com/2919625/106747905-caab1e80-6624-11eb-972e-86af7083de77.png">
in that case it didn't work even before the update

Reply:
does it work with the delay?